src/geo/leaflet/leaflet_cartodb_layergroup.js

```diff
@@ -142,6 +142,9 @@ L.CartoDBGroupLayerBase = L.TileLayer.extend({
     });
   },
 
+  getAttribution: function() {
+    return cdb.core.sanitize.html(this.options.attribution);
```

Comment:
Yeah, probably this is the most important addition, because leaflet adds the attribution when the layer is added ([in the `onAdd` method](https://github.com/CartoDB/cartodb.js/blob/develop/vendor/leaflet.js#L8131)), and it uses this method `getAttribution`.